ajax/libs/tesseract.js/package.json

```diff
@@ -0,0 +1,31 @@
+{
+  "name": "tesseract.js",
+  "filename": "tesseract.min.js",
+  "version": "1.0.10",
+  "description": "Pure Javascript Multilingual OCR",
+  "author": "antimatter15",
```

Comment:
The real name will be better.

Reply:
We don't need real name, and how can you know if it's real name or not?
It's better to use what it displays on GitHub page.

Reply:
https://github.com/antimatter15
Doesn't `Kevin Kwok` sound better?

Reply:
fine